### ISSUE TITLE
PersistentHandleReconnectTestCaseS0

### ISSUE THREAD

**Issue by TesterTester111** [Microsoft/WindowsProtocolTestSuites#119]
Hi,

Test PersistentHandleReconnectTestCaseS0 fails on validating 
_comment: This step would have covered "[MS-SMB2] 3.3.5.9.12: Handling the SMB2_CREATE_DURABLE_HANDLE_RECONNECT_V2 Create Context", "[MS-SMB2] The server MUST look up an existing open in the GlobalOpenTable by doing a lookup with the FileId.Persistent portion of the create context.", "[TestInfo] Open does not exist.", "[MS-SMB2] If the lookup fails, the server SHOULD<276> fail the request with STATUS_OBJECT_NAME_NOT_FOUND and proceed as specified in "Failed Open Handling" in section 3.3.5.9.", "[TestTag] InvalidIdentifier", "[TestInfo] The SUT platform is WindowsServer2008." 2. event parameters do not match comment: checking step 'event OpenResponse(STATUS_OBJECT_NAME_NOT_FOUND,durableHandleResponseContext,leaseResponseContext,c1)' assert failed: expected 'STATUS_OBJECT_NAME_NOT_FOUND', actual 'STATUS_SUCCESS' (status of OpenResponse, state S50)_

As seen in capture persistent part of FileID on reconnect request is the same as returned by server earlier, so validation mentioned above is supposed to pass.

There are another 3 cases with the same flow:
PersistentHandleReconnectTestCaseS285
PersistentHandleReconnectTestCaseS555
PersistentHandleReconnectTestCaseS588

Attached log and capture (.pcapng as .png).

[PersistentHandleReconnectTestCaseS0.txt](https://github.com/Microsoft/WindowsProtocolTestSuites/files/1197110/PersistentHandleReconnectTestCaseS0.txt)
![persistenthandlereconnecttestcases0](https://user-images.githubusercontent.com/25079571/28920974-c8d9ec44-785c-11e7-8782-384a1b645fe8.png)

Thanks,
Lilia

**Comment by yazeng** [comment]
Hi Lilia,

I looked into the capture. In the first Negotiate Request, the client does not set the persistent handle in Capabilities field. The server should not return the Negotiate Response with persistent handle set in the server capabilities. Even the client is asking for a persistent handle on the first connection, the server should not grant it to client. That's why our case expects 'STATUS_OBJECT_NAME_NOT_FOUND' instead of "STATUS_SUCCESS" in the second create request.

I attached the capture when running against Windows server.
[PersistentHandleReconnectTestCaseS0.zip](https://github.com/Microsoft/WindowsProtocolTestSuites/files/1206801/PersistentHandleReconnectTestCaseS0.zip)

Unzip it before you look into it.

Thanks,
Yan
